frontend/src/map/GameMap.js

```diff
@@ -1,14 +1,71 @@
+function tileKey(x, y, z) {
+    return [x, y, z].join(',');
+}
+
+function cellKey(x, y) {
+    return [x, y].join(',');
+}
+
 class GameMap {
-    constructor(width, depth, height) {
-        this.width = width
-        this.depth = depth
-        this.height = height
+    constructor() {
+        // For simplicity, assume for now that the minimum value for each
+        // dimension is 0.
+        this.maxX = 0;
+        this.maxY = 0;
+        this.maxZ = 0;
+
+        this.mapData = new Map();
```

Comment:
I like this change, but I'm getting [semantic satiation](https://en.wikipedia.org/wiki/Semantic_satiation) from the word map. Can we change `mapData` to `tileData` (if that makes sense)?

Reply:
Sure, no problem